component/rbac.libsonnet

```
@@ -6,6 +6,27 @@ local inv = kap.inventory();
 
 local params = inv.parameters.openshift4_authentication;
 
+local sudoGroups =
+  if params.sudoGroupName != null && params.sudoGroupName != '' then
+    local ln = std.trace(
+      (
+        '\nParameter `sudoGroupName` is deprecated.\n' +
+        'Please update your config to use `sudoGroups` instead.'
+      ),
+      params.sudoGroupName
+    );
+    com.renderArray([ ln ] + params.sudoGroups)
+  else
```

I'd probably write this as follows, but I'm fine with either implementation.

```suggestion
  local legacyGroup = 
    if params.sudoGroupName != null && params.sudoGroupName != '' then
      std.trace(
        (
          '\nParameter `sudoGroupName` is deprecated.\n' +
          'Please update your config to use `sudoGroups` instead.'
        ),
        [ params.sudoGroupName ]
      )
    else
      [];
  com.renderArray(legacyGroup + params.sudoGroups);
```